app/Command/Docs.hs

```diff
@@ -40,8 +43,12 @@ docgen (PSCDocsOptions fmt inputGlob) = do
   fileMs <- parseAndConvert input
   let ms = D.primModules ++ map snd fileMs
   case fmt of
-    Etags -> mapM_ putStrLn $ dumpEtags fileMs
-    Ctags -> mapM_ putStrLn $ dumpCtags fileMs
+    Etags -> do
+      let outputFilename = "TAGS" -- TODO: make this configurable
+      writeTagsToFile outputFilename $ dumpEtags fileMs
+    Ctags -> do
+      let outputFilename = "tags" -- TODO: make this configurable
```

Comment:
I don’t think we need a variable binding for this now, likewise in the etags branch.